it-and-security/lib/macos-device-health.policies.yml

```diff
@@ -10,7 +10,7 @@
   description: This policy checks if Firewall is enabled.
   resolution: An an IT admin, deploy a macOS, Firewall profile with the EnableFirewall option set to true.
   platform: darwin
-- name: macOS - Disable guest account
+- name: macOS - Guest user account is disabled by profile
```

Comment:
This policy was renamed to "Guest user account is disabled by profile" since it does not really check the status of Guest User.

Reply:
Should this be changed to match the new approach used in `cis-policy-queries.yml` files?

Reply:
You are changing the query name, shouldn't we change the query to use the plist approach used in the cis queries?

Reply:
I mean using the exact new version of the CIS policies and not renaming the policy:
```sh
SELECT 1 FROM plist WHERE path='/Library/Preferences/com.apple.loginwindow.plist' AND key='GuestEnabled' AND value = 0;
```

Reply:
I'll defer to @noahtalerman with this question.

Noah, to wit:
Do we want to allow the user to use both queries (check if profile installed AND check if GuestUser is allowed) ?
Or just check the end result (is Guest User allowed)?

Reply:
> using the exact new version of the CIS policies and not renaming the policy

I think let's do this instead.

FYI I opened a PR for this here: https://github.com/fleetdm/fleet/pull/17586

Please feel free to update or close mine.